pkg/lib/queueinformer/resourcequeue.go

```diff
@@ -39,6 +39,11 @@ func (r *ResourceQueueSet) RequeueEvent(namespace string, resourceEvent kubestat
 	r.mutex.RLock()
 	defer r.mutex.RUnlock()
 
+	if queue, ok := r.queueSet[metav1.NamespaceAll]; len(r.queueSet) == 1 && ok {
```

Comment:
Why is this necessary? if `""` is passed as the namespace, the following block will requeue in the "allnamespace" queue

Reply:
because we also want all of the requeues that specifically mention a namespace to go to the all-namespace queue as well.